Dockerfile

```diff
@@ -37,6 +37,7 @@ ENTRYPOINT ["./entrypoint"]
 FROM base AS no-build
 
 COPY --from=build-image /opt /opt
+COPY --from=build-image /usr/local /usr/local
```

Comment:
Don't we have scripts here which are only relevant for the image build phase? :thinking: 
Like `lnls-get-n-unpack`

Reply:
You are right. But after-build stages wouldn't be aware of such things. We must remove everything (including `lnls-get-n-unpack`) beforehand if we don't want it to be in the final image, otherwise we won't shrink the image size (which is related to #19).

Reply:
So it makes sense to wait for #59 before merging this. What do you think?

Reply:
I'm okay with it.